styles/globals.css

```diff
@@ -10,7 +14,7 @@
 
 body {
   font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
-  font-size: 14px;
+  font-size: 16px;
```

Comment:
use rems than px 
https://www.joshwcomeau.com/css/interactive-guide-to-flexbox/